_targets/build.common.imxrt

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+#
+# Common shell script's functions for imxrt
+#
+# Copyright 2021 Phoenix Systems
+# Author: Hubert Buczynski
+#
+[ "${BASH_SOURCE[0]}" -ef "$0" ] && echo "You should source this script, not execute it!" && exit 1
+
+
+# Function gets the following argument:
+# - $1 - array contains user's commands
+#
+# Function needs external variables to be defined in a project file:
+# - MAGIC_USER_SCRIPT - number which is assigned to user script
+# - NAME_USER_SCRIPT - name of the user script
+# - KERNEL_OFFS - offset on flash memory where kernel is located
+# - SIZE_PAGE
+# - PAGE_MASK
+b_mkscript_user() {
+	PROGS=()
+	cmds=("$@")
+	mkdir -p "$PREFIX_BUILD/plo/"
+	rm -f "$PREFIX_BUILD/plo/$NAME_USER_SCRIPT"
+
+	ksz=$((($(wc -c < "${PREFIX_PROG_STRIPPED}phoenix-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.elf") + SIZE_PAGE - 1) & PAGE_MASK))
+	poffs=$((KERNEL_OFFS + ksz))
+
+	{
+		printf "%08x\n" "$MAGIC_USER_SCRIPT"
+		for cmd in "${cmds[@]}"; do
```

Comment:
Whether the `cmds` variable is really needed.

Reply:
Currently, the commands array is passed as reference so that I think that this variable should be used.